lib/membrane_raw_audio.ex

```diff
@@ -228,79 +227,79 @@ defmodule Membrane.RawAudio do
   end
 
   @doc """
-  Returns minimum sample value for given format.
+  Returns minimum sample value for given sample format.
 
   Inlined by the compiler.
   """
   @spec sample_min(t) :: number
-  def sample_min(%__MODULE__{format: format}) do
+  def sample_min(%__MODULE__{sample_format: format}) do
     use Bitwise
 
-    case Format.to_tuple(format) do
+    case SampleFormat.to_tuple(format) do
       {:u, _, _} -> 0
       {:s, size, _} -> -(1 <<< (size - 1))
       {:f, _, _} -> -1.0
     end
   end
 
   @doc """
-  Returns maximum sample value for given format.
+  Returns maximum sample value for given sample format.
 
   Inlined by the compiler.
   """
   @spec sample_max(t) :: number
-  def sample_max(%__MODULE__{format: format}) do
+  def sample_max(%__MODULE__{sample_format: format}) do
     use Bitwise
```

Comment:
`use Bitwise` (line below) will be deprecated in Elixir 1.14, let's prepare and change it (not only here ofc):
```suggestion
    import Bitwise
```